src/scales/interpolatedColorScale.ts

```diff
@@ -12,7 +11,7 @@ export module Scales {
    * By default it generates a linear scale internally.
    */
   export class InterpolatedColor extends Scale<number, string> {
-    private static _COLOR_SCALES: ColorGroups = {
+    public static COLOR_SCALES = {
```

Comment:
This should probably be named `COLOR_RANGES` to match up with what it's called in the constructor. Maybe the fields inside should be all-caps too.

Alternatively, just make each range a separate `static` field.